resource-managers/kubernetes/core/src/test/scala/org/apache/spark/scheduler/cluster/k8s/KubernetesClusterSchedulerBackendSuite.scala

```diff
@@ -131,6 +131,22 @@ class KubernetesClusterSchedulerBackendSuite extends SparkFunSuite with BeforeAn
       pollEvents)
   }
 
+  test("executor's configmap.metadata.namespace set to conf.namespace otherwise default") {
+    assert(schedulerBackendUnderTest.createConfigMap().getMetadata.getNamespace === "default")
+
+    sparkConf.set(KUBERNETES_NAMESPACE, "namespace")
```

Comment:
better to create new sparkConf to avoid the impact on potential followup testcase